openpype/hosts/maya/plugins/publish/extract_camera_alembic.py

```diff
@@ -64,9 +69,10 @@ def process(self, instance):
 
                 # if baked, drop the camera hierarchy to maintain
                 # clean output and backwards compatibility
-                camera_root = cmds.listRelatives(
-                    camera, parent=True, fullPath=True)[0]
-                job_str += ' -root {0}'.format(camera_root)
+                camera_roots = cmds.listRelatives(
+                    cameras, parent=True, fullPath=True)
+                for camera_root in camera_roots:
+                    job_str += ' -root {0}'.format(camera_root)
```

Comment:
FYI - Note that this will fail if one camera happens to be a parent of the other since they'd have ancestor relationships.